content/articles/url-record.markdown

```diff
@@ -24,3 +24,36 @@ Enter the hostname you would like to redirect from, and the URL to redirect to.
 ## Redirecting over SSL
 
 Right now it is not possible for our redirection service to handle SSL requests for your domain names. Take a look at the article [Can a URL record redirect requests over SSL?](/articles/url-redirect-ssl) for additional information and an explanation of alternatives.
+
+## URL to URL redirects
+
+URL to URL redirects can't be done with our URL record. Only domains or subdomains can be redirected to complete URLs. The following redirect will not work:
+
+    foo.com/blog to bar.com/blog
+
+## Wildcard URL redirects
+
+You can configure a wildcard redirect using the same conventions of a DNS wildcard record. However, the target cannot contain any wildcard reference.
+
+    # Valid redirects
+    *.foo.com to bar.com
+    *.foo.com to bar.com/path
+
+    # Invalid redirects
+    *.foo.com to *.bar.com
+
+The wildcard pattern can only be on the left outermost level.
+
+    # Valid redirects
+    *.foo.com to bar.com
+
+    # Invalid redirects
+    foo.*.com to bar.com
```

Comment:
It might be better to use an example like `foo.*.bar.com` since no one has access to the full `com` zone from DNSimple. :wink: